src/openrct2/ride/VehicleSubpositionData.cpp

```diff
@@ -21424,6 +21424,2469 @@ CREATE_VEHICLE_INFO(TrackVehicleInfo_9571BC, {
     {     16,  -63,    0, 24,  0,  0 }, {     16,  -64,    0, 24,  0,  0 },
 })
 
+namespace LeftLane
+{
+    CREATE_VEHICLE_INFO(TrackVehicleInfoLeftQuarterTurn5Tiles0, {
+        {     31,    9,    0,  0,  0,  0 },{     30,    9,    0, 31,  0,  0 },{     29,    9,    0, 31,  0,  0 },{     28,    9,    0, 31,  0,  0 },{     27,    9,    0, 31,  0,  0 },
```

Comment:
Please change the formatting to match other definitions, especially w/r/t comma placement between the }{.